scripts/svm/utils/constants.ts

```diff
@@ -1,4 +1,12 @@
+import BN from "bn.js";
+
 export const CIRCLE_IRIS_API_URL_DEVNET = "https://iris-api-sandbox.circle.com";
 export const CIRCLE_IRIS_API_URL_MAINNET = "https://iris-api.circle.com";
 export const SOLANA_USDC_MAINNET = "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v";
 export const SOLANA_USDC_DEVNET = "4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU";
+export const SEPOLIA_CCTP_MESSAGE_TRANSMITTER_ADDRESS = "0x7865fAfC2db2093669d92c0F33AeEF291086BEFD";
+export const MAINNET_CCTP_MESSAGE_TRANSMITTER_ADDRESS = "0x0a992d191deec32afe36203ad87d7d289a738f81";
+export const SEPOLIA_USDC_ADDRESS = "0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238";
+export const MAINNET_USDC_ADDRESS = "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48";
```

Comment:
better reuse `TOKEN_SYMBOLS_MAP` as exported in `utils/constants.ts`

Reply:
Do you mean doing this?
https://github.com/across-protocol/contracts/pull/786/commits/c17299c65184e069af3dfda37afa39e61e9c1e78

Reply:
kindof, but better avoid putting the var in constants and not pass chainId. E.g. can pull required token for required chain dynamically, like [here](https://github.com/across-protocol/contracts/blob/0785cf2519e52d40629c6d3337f4a7c3441f3c7d/scripts/svm/proposeRebalanceToSpokePool.ts#L51)

Reply:
You are absolutely right. I fixed it here: https://github.com/across-protocol/contracts/pull/786/commits/10a60f2b7d5f9bbd443fa8342df6f24b47b43e2b